python/paddle/v2/fluid/__init__.py

```diff
@@ -39,30 +39,16 @@
 import clip
 from memory_optimization_transpiler import memory_optimize
 import profiler
+import unique_name
 
 Tensor = LoDTensor
 
 __all__ = framework.__all__ + executor.__all__ + concurrency.__all__ + [
-    'io',
-    'initializer',
-    'layers',
-    'nets',
-    'optimizer',
-    'learning_rate_decay',
-    'backward',
-    'regularizer',
-    'LoDTensor',
-    'CPUPlace',
-    'CUDAPlace',
-    'Tensor',
-    'ParamAttr',
-    'WeightNormParamAttr',
-    'DataFeeder',
-    'clip',
-    'SimpleDistributeTranspiler',
-    'DistributeTranspiler',
-    'memory_optimize',
-    'profiler',
+    'io', 'initializer', 'layers', 'nets', 'optimizer', 'learning_rate_decay',
```

Comment:
It's better to add a `,` after `'unique_name'`, because the format of this snippet code always changes.